console/src/main/scala/io/joern/console/cpgcreation/ImportCode.scala

```diff
@@ -103,6 +124,28 @@ class ImportCode[T <: Project](console: io.joern.console.Console[T])(implicit
     def isAvailable: Boolean =
       cpgGeneratorForLanguage(language, config.frontend, config.install.rootPath.path, args = Nil).exists(_.isAvailable)
 
+    def fromDirectory(inputPath: String, projectName: String = "", args: List[String] = List()): Cpg = {
```

Comment:
given that it's so related and already in the same file, would it make sense to have the two `fromDirectory` and `fromFile` directly next to each other?

Reply:
Well, we need them at `ImportCode` and `Frontend` so that e.g., both `importCode.fromFile` and `importCode.jssrc.fromFile` work.